backend/src/helpers/VeridaHelper.ts

```diff
@@ -226,13 +121,13 @@ export default class VeridaHelper {
    */
 
   private static async init(issuerPrivateKey: string) {
-    const context = Network.connect({
-      context: {
-        name: VERIDA_APP_NAME,
-      },
+
+
+    const context = await Network.connect({
       client: {
-        environment: VERIDA_ENVIRONMENT,
+        environment: EnvironmentType.TESTNET,
```

Comment:
Better to use VERIDA_ENVIRONMENT. Why do we need to hardcode this?

Reply:
@cmcWebCode40 Fix this and I think I can merge this.

Reply:
Unfortunately I need to deploy this now. I've raised a new issue for this.